versioneer.py

```diff
@@ -1723,7 +1722,7 @@ def do_setup():
     root = get_root()
     try:
         cfg = get_config_from_root(root)
-    except (EnvironmentError, configparser.NoSectionError, configparser.NoOptionError) as e:
+    except (OSError, configparser.NoSectionError, configparser.NoOptionError) as e:
```

Comment:
Change of error from EnvironmentError to OSError. According to the [Built-in Exceptions docs](https://docs.python.org/3/library/exceptions.html#OSError.filename2), EnvironmentError was merged into OSError in 3.3